conjure-go-client/httpclient/client_params.go

```diff
@@ -473,6 +473,14 @@ func WithRefreshableConfig(config RefreshableClientConfig) ClientParam {
 			}
 			return *duration
 		}))
+		b.Timeout = refreshable.NewDuration(config.Map(func(i interface{}) interface{} {
```

Comment:
change the `Map` to `MapClientConfig` then you don't need to do the type cast below